pages/02-starting-out.md

```diff
@@ -26,7 +26,7 @@ basic computations.
 import Html
 
 main =
-  Html.text (toString toPrint)
+  Html.text (Debug.toString toPrint)
```

Comment:
I think `fromInt` would be a better fit here (and in the example below as well).